### PR TITLE
Document formats of time columns in main_summary

### PR DESCRIPTION
**Pull request by tdsmith**
These are from [a spreadsheet](https://docs.google.com/spreadsheets/d/1DQk0YQx2PLaY2ZMTdhF2TA-ueZ6g5PWwUqbPWL90AHM/edit#gid=0) that @acmiyaguchi shared in #fx-metrics; is this a good place to have this information?

### CODE REVIEW

**Comment by tdsmith** [comment]
Oh, I see this is #122; I don't know if you were anticipating something more than just copy-pasting the spreadsheet into Markdown, but here's a start, anyway :)

File: datasets/batch_view/main_summary/reference.md [hunk 1]
```diff
@@ -452,6 +452,21 @@ Most of the fields are simple scalar values, with a few notable exceptions:
...
 
+## Time formats
+
+Columns in `main_summary` may use one of a handful of time formats with different precisions:
+
+Column Name              | Origin                                   | Description                                  | Example                         | Spark                                                                    | Presto
+-------------------------|------------------------------------------|----------------------------------------------|---------------------------------|--------------------------------------------------------------------------|---------------------------------------------------------------------------
+`timestamp`              | stamped at ingestion                     | nanoseconds since epoch                      | `1504689165972861952`           | `from_unixtime(timestamp/1e9)`                                           | `from_unixtime(timestamp/1e9)`
+`submission_date`        | derived from timestamp                   | `YYYYMMDD` date string of timestamp          | `20170906`                      | `from_unixtime(unix_timestamp(submission_date, 'yyyyMMdd'))`             | `date_parse(submission_date, '%Y%m%d')`
```
Comment (mreid-moz) [inline datasets/batch_view/main_summary/reference.md]: drive-by: please change this to `submission_date_s3` - it is the same format, but is preferred since it is a partitioning column and will result in faster querying.

Comment (tdsmith) [reply datasets/batch_view/main_summary/reference.md]: :+1:; thanks!
